README.md

```diff
@@ -1034,6 +1034,14 @@ All statements above will return false if used with `===`
 
 <br/><br/><br/>
 
+## ![✔] 7.2. Replicate the Node process
+
+ **TL;DR:** If you really need to use some heavy computational work in your application and you can’t make use of the [non-blocking event loop](https://nodejs.org/en/docs/guides/dont-block-the-event-loop), and if you have a managed server with more than a single CPU, you should consider using the native Node.js `cluster` module or a library like [PM2](http://pm2.keymetrics.io/docs/usage/cluster-mode). Cluster mode will allow you to take advantage of your CPU resources to replicate the single thread when the Event Loop is blocked.
+
+**Otherwise:** In high peak traffic your users could suffer some latency when the Event Loop is blocked and if you have a multi CPU managed server you will be wasting valuable resources.
```

Comment:
Sums it up well, consider spicing it up - You pay for 4 cpu and starve 3 of them.